concourse/scripts/test_postgresql-hll.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -l
+
+set -exo pipefail
+
+function activate_standby() {
+    gpstop -may -M immediate
+    export PGPORT=6001
+    export MASTER_DATA_DIRECTORY=/home/gpadmin/gpdb_src/gpAux/gpdemo/datadirs/standby
+    gpactivatestandby -ad $MASTER_DATA_DIRECTORY
+}
+
+function _main() {
+    tar -xzf /home/gpadmin/bin_postgresql-hll/postgresql-hll-*_x86_64.tar.gz -C /usr/local/greenplum-db-devel
```

Comment:
I suggest call `install_gpdb_component` here, then at least we have a chance to test that script.